src/Task1/React.CreateElement.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const CreateElement = () => {
+  // Creating react element using createElement
+  const title = React.createElement("h1", {}, "My First React Code");
+  const paragraph = React.createElement(
+    "p",
+    {},
+    "Writing component using createElement"
+  );
+  const reactCreateElement = React.createElement(
+    "div",
+    { style: { backgroundColor: "yellow" } },
+    [title, paragraph]
```

Comment:
When you use `createElement()`, children should be passed just as a series of values separated by a comma (`,`). No need to wrap into an array. In fact, arrays in this case will be treated as list nodes, and React will warn about specifying `key` for each item in the list. This is unwanted in cases like yours.

Reply:
I fixed it